backend/custom_models/migrations/0011_upload_background_markdown.py

```diff
@@ -0,0 +1,368 @@
+# Generated by Django 2.1.7 on 2019-05-30 14:07
+# Edited by Francis Normand to upload the background markdown content
```

Comment:
...for the sample test. <- I think it's important to make this specific so it doesn't get confused with Pizza or otherwise.

Reply:
Added in commit #https://github.com/code-for-canada/project-thundercat/pull/400/commits/94c24e50e97b5a820ca6f70f38fafc74f9c4b42b